src/properties.jl

```diff
@@ -253,6 +253,21 @@ function class_setproperty!(::Type{ObjectCreateProperties}, p::Properties, name:
     class_setproperty!(superclass(ObjectCreateProperties), p, name, val)
 end
 
+function get_track_order(p::Properties)
+    attr = Ref{UInt32}()
+    API.h5p_get_attr_creation_order(p, attr)
```

Comment:
It would make sense to define in https://github.com/JuliaIO/HDF5.jl/blob/master/src/api/helpers.jl 
```julia
function h5p_get_attr_creation_order(p)
    attr = Ref{UInt32}()
    h5g_get_num_objs(p, attr)
    return attr[]
end
```
same for `h5p_get_link_creation_order`

Reply:
Thanks for pointing `src/api/helpers.jl`, I've added the helpers for both.